File: java-sdk/radar-schemas-core/src/main/java/org/radarbase/schema/specification/active/questionnaire/QuestionnaireSource.java
package org.radarbase.schema.specification.active.questionnaire;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;
import org.radarbase.schema.specification.active.ActiveSource;

@JsonInclude(Include.NON_NULL)
public class QuestionnaireSource extends ActiveSource<QuestionnaireDataTopic> {
}

File: java-sdk/radar-schemas-core/src/main/java/org/radarbase/schema/specification/active/questionnaire/QuestionnaireSource.kt
package org.radarbase.schema.specification.active.questionnaire

import com.fasterxml.jackson.annotation.JsonInclude
import com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL
import org.radarbase.config.OpenConfig
import org.radarbase.schema.specification.active.ActiveSource

@JsonInclude(NON_NULL)
@OpenConfig
class QuestionnaireSource : ActiveSource<QuestionnaireDataTopic>()
